static/app/components/replays/breadcrumbs/replayTimelineSpans.tsx

```diff
@@ -72,20 +73,21 @@ const Spans = styled('ul')`
   margin: 0;
   padding: 0;
 
-  height: ${space(3)};
+  height: ${space(1.5)};
```

Comment:
We could replace the `top: -3px` inside each span and put it up here as a margin-bottom. Changed from `3` to `space(0.5)` (aka `4px`) for fewer magic numbers.

```suggestion
  height: ${space(1.5)};
  margin-bottom: ${space(0.5)};
```

I think it's good to move css properties up into parents whenever possible, because then instead of N `li` elements each getting the rule applied, it's just 1 `ul` that needs to do the work.